Gemfile

```diff
@@ -3,6 +3,7 @@ source "https://rubygems.org"
 # back-end
 gem "trollop", "2.1.3"
 gem "bicho", "0.0.16"
+gem "gitlab", "4.5.0"
```

Comment:
Why are you not using the current version `4.7.0` of the gitlab gem? I quickly tried it and it seems to work just fine.

Reply:
Well, it was 4.5.0 back when I started. Didn't check for updates yes. 
I suppose if it works I will just bump it to the current version.